workshop_operations/workshop-guide.md

```diff
@@ -43,7 +43,7 @@ Table of Contents
 
 ## A Week Before
 
-* obtain and send pre-workshop questionnaire to attendants
+* ask local coordinator to obtain and send pre-workshop questionnaire to attendants
```

Comment:
Is this about Carpentries' pre-workshop survey that is linked from the workshop website? Instructors will receive an email with links to the pre-/post-survey results from Talisha (as long as they are registered in AMY, based on self-organized workshop request form).

Reply:
I see. The problem is that the instructors don#t have the participants' email addresses, so I think this is a task that the coordinator(s) have to do somehow.
Let me think a bit about how this could be done smoothly.

Reply:
Thanks, Annika! So you meant about the Carpentries pre-workshop survey. I think we can send a batch email to all the participants a few days before to remind them to answer the survey linked from the workshop website. Especially for workshops with a long waiting list, we used to send a reminder a few days before the workshop including

- instructors' message inc. necessary set up (often based on instructor's email to us; we ask instructors to send us an email that they want to communicate with participants)
- asking signing off if they know that they cannot come 

so here we can add a reminder of workshop survey? 
In that sense, probably what the lead instructor should do "a week before" is like the following?

- Send local coordinator an email with message that you want to communicate with the participants (local coordinator will forward the message to all the participants)

Reply:
I think that would work!